README.md

```diff
@@ -144,6 +144,21 @@ The control mapping on Wii VC will be the following:
 
 Note that you can use the Wii Classic Controller too.
 
+## Using GDB
+### How to setup
+You will need a nightly build of [Ares](https://github.com/ares-emulator/ares)
+
+- For Linux/WSL you need to install ``gdb-multiarch`` and [this](https://pypi.org/project/gdb-tools/) Python module, then use the launch configuration ``Ares GDB (Linux)``. If using WSL, you might need to disable the IP change for WSL and update it in the launch configuration, also you might need to add a rule in the firewall to allow traffic. If you're on native Linux simply make the address ``localhost:PORT`` and it should work (port being the one shown on Ares).
```

Comment:
what is https://pypi.org/project/gdb-tools/ for?

Reply:
iirc it was for one of the two python scripts, but idk if they have been updated to remove that requirement since (unless it was never required and I got baited by an unused import)

Reply:
afaik it's not needed for either script